src/core/Notification/BaseNotification/BaseNotification.baseStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseNotificationBaseStyles = (theme: SuomifiTheme) => css`
+  &.fi-notification {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    p {
+      font-size: 16px;
+    }
+    & .fi-notification_label {
+      ${font(theme)('bodySemiBold')}
+      margin-top: -4px;
+      margin-bottom: ${theme.spacing.xxs};
+    }
+
+    padding-bottom: 17px;
+    width: 100%;
+    & .fi-notification_style-wrapper {
+      display: flex;
+      align-items: flex-start;
+    }
+    & .fi-notification_icon-wrapper {
+      margin-top: 17px;
+      & .fi-notification_icon {
+        height: 24px;
+        width: 24px;
+      }
+    }
+    & .fi-notification_text-content-wrapper {
+      display: flex;
+      flex: 3 1 auto;
+      flex-direction: column;
+      padding: 9px 20px 0 20px;
+      margin-top: ${theme.spacing.s};
+      margin-bottom: 0;
+      & .fi-notification_content {
+        vertical-align: middle;
+        ${font(theme)('bodyTextSmall')}
+      }
+    }
+    box-shadow: 0px 4px 8px 0px rgba(41, 41, 41, 0.2);
+    border-radius: 4px;
+    /* Status variant styles */
+    &--neutral {
```

Comment:
If we end up keeping the two separate components, these styles should probably be divided among them according to which variants apply to which component.

Reply:
Now we have only one notification and Toast.